modules/web/src/com/haulmont/addon/ldap/web/screens/messages.properties

```diff
@@ -0,0 +1,73 @@
+matchingRuleScreenCaption = LDAP matching rule
+ldapSettingsScreenCaption = LDAP config
+matchingRuleTableConditionColumn = Condition
+matchingRuleTableCubaColumn = Cuba
+matchingRuleTableTypeColumn = Type
+matchingRuleTableOptionsColumn = Options
+matchingRuleTableStateColumn = Active
+ruleEditButton = Edit
+ruleRemoveButton = Remove
+simpleRuleCreateButton = Create simple rule
+caption = Screen
+testMatchingRuleScreen = Test matching Rule
+testRuleScreenTestButton = Test rules
+testRuleScreenRuleTable = Applied matching rules
+testRuleScreenRoleTable = Applied roles
+testMatchingRuleTableTypeColumn = Type
+testMatchingRuleTableOptionsColumn = Options
+testMatchingRuleTableResultColumn = Cuba
+testMatchingRuleTableConditionColumn = Condition
+testRuleScreenUserName = User login
+settingsScreenUrl = Url
```

Comment:
All abbreviations in capital letters: URL

Reply:
Done